x-pack/plugins/observability_solution/observability/public/components/custom_threshold/components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -154,6 +165,25 @@ export default function AlertDetailsAppSection({
   const annotations: EventAnnotationConfig[] = [];
   annotations.push(alertStartAnnotation, alertRangeAnnotation);
 
+  useEffect(() => {
+    setTimeRange(getPaddedAlertTimeRange(alertStart!, alertEnd));
```

Comment:
Should we check if `alertStart` is not undefined instead of forcing its type here?

Reply:
The idea is that since we are in alert details page, the alert start time is always there (or we won't have an alert). The alert end time, on the other hand, can be undefined if the alert is still active.